packages/evo-marko/src/tags/evo-3d-viewer/index.marko

```diff
@@ -0,0 +1,81 @@
+import type { AttrClass, AttrString } from "marko/tags-html";
+import type { ModelViewerElement } from "@google/model-viewer";
+import { load as modelViewerLoad } from "@internal/model-viewer";
+export interface Input extends Partial<ModelViewerElement> {
+    class?: AttrClass;
+    errorText?: AttrString;
+    a11yLoadText?: AttrString;
+    onLoadError?: (err: CustomEvent) => void;
+}
+<const/{
+    class: inputClass,
+    src,
+    errorText,
+    onLoadError,
+    a11yLoadText,
+    // ...htmlInput
+}=input/>
+
+<const/htmlInput=(({
+    class: inputClass,
+    src,
+    errorText,
+    onLoadError,
+    a11yLoadText,
+    ...rest
+}) => rest)(input)/>
+
+<let/ready=false/>
+<let/error=false/>
```

Comment:
Can we ever be ready and error at the same time? If not, this should probably be `<let/state="pending" as "ready" | "error" | "pending">`